src/colorpickr.js

```diff
@@ -322,37 +335,47 @@ class ColorPickr extends React.Component {
       modeInputs = (
         <div>
           <div
-            {...theme('inputModeContainer', `${channel === 'r' ? 'active' : ''}`)}
+            {...theme(
+              'inputModeContainer',
+              `${channel === 'r' ? 'active' : ''}`
+            )}
           >
             <ModeInput
               id="r"
               theme={themeModeInput}
               name={this.modeInputName}
               checked={channel === 'r'}
               onChange={this.setChannel}
+              readOnly={readOnly}
             />
             <RGBInput
               id="r"
               theme={themeNumberInput}
               value={r}
               onChange={this.changeRGB}
+              readOnly={readOnly}
             />
           </div>
           <div
-            {...theme('inputModeContainer', `${channel === 'g' ? 'active' : ''}`)}
+            {...theme(
```

Comment:
Why all the formatting this in this PR? We should scope changes to just adding a readonly/disabled prop